features/step_definitions/plans_steps.rb

```diff
@@ -181,6 +181,18 @@ def default_service
 end
 
 def select_default_plan(plan)
+  select_default_plan_by_name(plan.name)
+end
+
+def select_default_plan_by_name(name)
+  select = find(:css, '#default_plan_card .pf-c-select')
+  select.find(:css, '.pf-c-button.pf-c-select__toggle-button').click unless select[:class].include?('pf-m-expanded')
+  select.find('.pf-c-select__menu-item', text: name).click
```

Comment:
I think we could try fixing Codeclimate by using `within` instead of declaring `select`:
```suggestion
  within('#default_plan_card .pf-c-select') do
    find('.pf-c-button.pf-c-select__toggle-button').click unless select[:class].include?('pf-m-expanded')
    find('.pf-c-select__menu-item', text: name).click
  end
```
But I don't know how would be do this instead `select[:class].include?('pf-m-expanded')`. Maybe `self` reference to the current node or something.